components/iam/base/external-secrets/mtls-ca-validators.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: mtls-ca-validators
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    argocd.argoproj.io/sync-wave: "-1"
+spec:
+  dataFrom:
+    - extract:
+        conversionStrategy: Default
+        decodingStrategy: None
+        key: staging/group-sync/mtls-ca-validators
```

Comment:
we need to patch this one for the 2 production environments, should be "production/group-sync/mtls-ca-validators" for them